x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit/threshold_watch_edit.tsx

```diff
@@ -383,10 +384,19 @@ const ThresholdWatchEditUi = ({ intl, pageTitle }: { intl: InjectedIntl; pageTit
         <EuiSpacer />
         {shouldShowThresholdExpression ? (
           <Fragment>
+            <EuiTitle size="s">
```

Comment:
Can we use EuiHelpText placed below the expression instead of this error between the title and expression? I think this will follow the pattern of how we surface other field-level errors. And I think we can simplify the error message to "Your expression has errors."

![image](https://user-images.githubusercontent.com/1238659/58066805-b6b54580-7b3f-11e9-98ad-9513519f913a.png)

Reply:
Good suggestion.

I don't think the `EuiExpression` comes with help text out of the box. However, I went ahead and moved the `EuiText` that existed for this below the expression and reduced the font. Let me know what you think.

<img width="630" alt="Screen Shot 2019-05-21 at 8 36 45 AM" src="https://user-images.githubusercontent.com/5226211/58104284-8a86dc80-7bb2-11e9-9fa3-6abf4f40286a.png">

Reply:
Looks great to me!